### PR TITLE
Fix links

### PR DESCRIPTION
**Pull request by dnth**
## Describe changes
This PR fixes all broken links in docs and readme pages with the recent docs page update.

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/advanced-guide/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Other (add details above)

### CODE REVIEW

File: docs/book/developer-guide/stacks-profiles-repositories/repository.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ ZenML repository initialized at /tmp/zenml.
 The local active profile was initialized to 'default' and the local active stack
 to 'default'. This local configuration will only take effect when you're running
 ZenML from the initialized repository root, or from a subdirectory. For more
-information on profile and stack configuration, please visit https://docs.zenml.io.
+information on profile and stack configuration, please visit https://docs.zenml.io .
```
Comment (fa9r) [inline docs/book/developer-guide/stacks-profiles-repositories/repository.md:110]: Was this change intentional? If not, might want to undo:
```suggestion
information on profile and stack configuration, please visit https://docs.zenml.io.
```

**Review by fa9r — APPROVED** [review]
LGTM!

File: docs/book/developer-guide/stacks-profiles-repositories/repository.md (hunk 1 above)
Comment (dnth) [reply docs/book/developer-guide/stacks-profiles-repositories/repository.md:110]: Yes I though that looks a little awkward too.

**Review by htahir1 — APPROVED** [review]
